source/differences_v220_v231.rst

```diff
@@ -0,0 +1,243 @@
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Field Name
+     - Description
+   * - **acceptLanguage**
+     - Value representing the Browser language preference present in the HTTP header
+   * - **appIp**
+     - External IP address used by the 3DS Requestor App
+   * - **acquirerCountryCode**
+     - The code of the country where the acquiring institution is located
+   * - **acquirerCountryCodeSource**
+     - System setting the Acquirer Country Code
+   * - **broadInfo**
+     - Unstructured information sent between 3DS Server, DS and ACS
+   * - **cardSecurityCode**
+     - Three- or four-digit security code printed on the card
+   * - **deviceBindingStatus**
+     - Communication of Device Binding Status between ACS, DS and 3DS Requestor
+   * - **deviceBindingStatusSource**
+     - System setting Device Binding Status
+   * - **deviceId**
+     - Unique and immutable identifier linked to a device
+   * - **defaultSdkType**
+     - Characteristics of a Default-SDK
+   * - **multiTransaction**
+     - Additional transaction information for multiple transactions or Merchants
+   * - **payeeOriginAReq**
+     - Origin of the payee for SPC Transaction Data
+   * - **payTokenInfo**
+     - Information about detokenised Payment Token
+   * - **recurringAmount**
+     - Recurring amount in minor units of currency
+   * - **recurringCurrency**
+     - Currency in which the Recurring Amount is expressed
+   * - **recurringDate**
+     - Effective date of the new authorised amount
+   * - **recurringExponent**
+     - Minor units of currency for recurring amount
+   * - **recurringInd**
+     - Indicates whether recurring/instalment payment has fixed or variable amount and frequency
+   * - **sdkServerSignedContent**
+     - JWS object created by the Split-SDK Server for AReq message
+   * - **sdkSignatureTimestamp**
+     - Date and time when 3DS SDK generated the Split-SDK Server Signed Content
+   * - **sdkType**
+     - Indicates the type of 3DS SDK
+   * - **sellerInfo**
+     - Additional transaction information for marketplace transactions
+   * - **spcIncompInd**
+     - Reason that SPC authentication was not completed
+   * - **splitSdkType**
+     - Characteristics of a Split-SDK
+   * - **taxId**
+     - Cardholder's tax identification
+   * - **threeDSMethodId**
+     - 3DS Server Transaction ID used during previous 3DS Method execution
+   * - **threeDSRequestorSpcSupport**
+     - Indicates if 3DS Requestor supports SPC authentication
+   * - **userId**
+     - Identifier of the transacting user's Browser Account ID
+
+Fields Removed in v2.3.1
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Field Name
+     - Description
+   * - **threeDSReqAuthMethodInd**
+     - Signature verification performed by DS on cardholder authentication mechanism
+   * - **whiteListStatus**
+     - Renamed to ``trustListStatus``
+   * - **whiteListStatusSource**
+     - Renamed to ``trustListStatusSource``
```

Comment:
`whiteListStatus` and `whiteListStatusSource` are also under renamed, so maybe they dont need to be under removed?

Reply:
I removed it here https://github.com/clearhaus/3DSv2-api-documentation/pull/37/commits/84737b33481d46e6e68e93aca7b9aae15304a1e4